src/accounting/helpers.ts

```diff
@@ -0,0 +1,47 @@
+import { Map as IMap, List } from 'immutable';
+import moment from 'moment';
+import { BigNumber } from 'bignumber.js';
+
+import { InvalidParamError } from '../errors';
+
+export const transactionIsBuy = (transaction: IMap<any, any>) => {
```

Comment:
Is there a way to type the keys of an IMAP?

Reply:
Yeah this is the wrong type to be using. I'm going to remove it and see if typescript will infer the type. If it can't, then i'll use the `Transaction` type defined in types.ts. good catch.

Reply:
added an `ITransaction` type.